CONTRIBUTING.md

```diff
@@ -1,3 +1,58 @@
+# Contributing
+
+## Getting started
+
 The Contributing page in the wiki provides some guidance on how to get started developing Uyuni.
 
 https://github.com/uyuni-project/uyuni/wiki/Contributing
+
+## Pull Requests
+
+The following roles exist in the context of a Pull Request:
+
+- Creator
+- Reviewer
+- Collaborators
+
+The guidelines noted here describe the expectations that should be met when submitting a Pull request.
+
+### Pull Request Creators
```

Comment:
Even though it's in the PR template, I would emphasize here to contribute to unit tests with best effort.
Maybe some item like this:
```
- Different parts of the code has different test infrastructures (JUnit for Java, Jest for JS, etc.). Please make sure to introduce appropriate tests for any new functionality. Pull requests that introduce untested code won't be approved.
```

Reply:
Thanks all for the suggestions! :+1: 

I just reworked this note to make it a bit shorter and added it to the PR.